config/envs/default.py

```diff
@@ -279,3 +279,6 @@ class DefaultConfig(object):
     REDIS_MLINKS_URL = REDIS_INSTANCE_URI + "/0"
     REDIS_SESSIONS_URL = REDIS_INSTANCE_URI + "/1"
     SESSION_REDIS = redis.from_url(REDIS_SESSIONS_URL)
+
+    # Connexion
+    CONNEXION_OPTIONS = {}
```

Comment:
Do we want to remove swagger from all deployed envs, or just production? It does feel like a semi-useful thing, and I don't think I would hate it existing in our non-prod envs. This may be mostly because I can't use authenticator locally, though, so maybe once I've solved that issue the need for deployed authenticator swagger docs diminishes...

Reply:
Agreed to add back in to dev env, but only after we upgrade connexion